src/app/@ansyn/ansyn/app-effects/effects/status-bar.app.effects.ts

```diff
@@ -93,7 +90,7 @@ export class StatusBarAppEffects {
 	onCancelGeoFilter$ = this.actions$.pipe(
 		ofType<UpdateGeoFilterStatus>(StatusBarActionsTypes.UPDATE_GEO_FILTER_STATUS),
 		filter(action => action.payload === undefined),
-		withLatestFrom(this.store.select(selectRegion)),
+		withLatestFrom(this.store.select(selectGeoFilterStatus)),
```

Comment:
it should be taken from the region.
to see why,
1. draw a polygon search.
2. check the point search
3. click on the menu.

the polygon search redraw on the map, but in the geoFilter will be write `Point`